packages/flutter/lib/src/painting/binding.dart

```diff
@@ -79,6 +81,45 @@ mixin PaintingBinding on BindingBase, ServicesBinding {
     super.evict(asset);
     imageCache.clear();
   }
+
+  /// Listenable for OSes system fonts change event.
+  ///
+  /// System fonts can change when system installs or removes new font. To
+  /// correctly reflect the change, it is important to relayout text related
+  /// widgets when this happens.
```

Comment:
Maybe add a little more details here, e.g. objects that show text and/or measure text (e.g. via [TextPainter]) should listen to this and redraw/remeasure.